packages/forklift-console-plugin/src/modules/Plans/views/details/components/PlanPageHeadings.tsx

```diff
@@ -56,6 +85,18 @@ export const PlanPageHeadings: React.FC<{ name: string; namespace: string }> = (
         key={'providerCriticalCondition'}
       />,
     );
+  } else if (preserveIpsWithPodMapCondition()) {
```

Comment:
no need for elseif you can add a return in the previous if condition

Reply:
This code section is not part of a function so I can't use return there and don't see a value in wrapping it in a function....

Reply:
Yes u right i thought it part of a function as this is very strange and non-performing, as array will build itself with every change in the file, not only for plans objs that arrive, when looking deeper on this file I can see the alerts its just an array without any state, it causes a lot of unnecessary rerenders.. this is not related to this pr but seems we have work he to do on MTV UI :)